src/main/java/io/github/apace100/apoli/power/factory/condition/BlockConditions.java

```diff
@@ -173,6 +173,8 @@ public static void register() {
                 BlockState state = block.getBlockState();
                 return ((Comparison)data.get("comparison")).compare(state.getBlock().getHardness(), data.getFloat("compare_to"));
             }));
+        register(new ConditionFactory<>(Apoli.identifier("powered"), new SerializableData(),
+            (data, block) -> block.getBlockEntity().getWorld().getReceivedRedstonePower(block.getBlockPos()) > 0));
```

Comment:
I think users would prefer it more if this block condition type were to have a `comparison` and a `compare_to` field. This way, it can check for power levels in an inclusive/exclusive way, not just if its greater than 0